netdata-installer.sh

```diff
@@ -343,7 +340,7 @@ while [ -n "${1}" ]; do
     "--disable-ebpf") NETDATA_DISABLE_EBPF=1 NETDATA_CONFIGURE_OPTIONS="${NETDATA_CONFIGURE_OPTIONS//--disable-ebpf/} --disable-ebpf" ;;
     "--aclk-ng") ;;
     "--aclk-legacy")
-      NETDATA_CONFIGURE_OPTIONS="${NETDATA_CONFIGURE_OPTIONS//--with-aclk-legacy/} --with-aclk-legacy"
+      fatal "ACLK legacy is no longer supported."
```

Comment:
What's the strategy we want to follow? Should we have one release with the PSA that support for ACLK legacy will be dropped, and in the following release drop it? If this is the case, I think it's better to simply remove `--aclk-legacy` option instead of `fatal`-ing.

Reply:
The plan is to list it as deprecated for this current release and then remove it after the release.

As far as handling of the option, we either provide a useful error message and exit (because we cannot do what the user is asking us to do), or we dump a usage message and exit due to an unrecognized option. The first approach is preferable because even though 99% of users affected by this will end up ignoring the error message and complain to us anyway, we can at least point at the error message and claim that we _did_ tell them what was wrong.